### PR TITLE
[stable/velero] Introduce crds directory for compatibility with Helm v3

### PR DESCRIPTION
**Pull request by jmapro**
#### What this PR does / why we need it:

* This adds `crds` directory which has all 10 crds
* Remove templating options from crds. Keep only `app.kubernetes.io/name` with fixed value.
* Add `templates/crds.yaml` to keep backward compatibility with Helm v2
* Update the chart version to 2.4.0

#### Which issue this PR fixes

  - Related to Issue #19008 

#### Special notes for your reviewer:

* This is not a complete migration to Helm v3. This PR make the Chart installable with Helm 3.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/velero/templates/crds.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{{- range $path, $bytes := .Files.Glob "crds/*.yaml" }}
+{{ $.Files.Get $path }}
+---
+{{- end }}
```
Comment (scottrigby) [inline stable/velero/templates/crds.yaml]: Just needs new line at EOF

**Review by scottrigby — CHANGES_REQUESTED** [review]

**Comment by nrb** [comment]
Thanks @jmapro! I haven't had a chance to look at what changes Helm 3 introduces, and will be on vacation next week. Hopefully @prydonius and the other maintainers will be able to take a look in the meantime.

File: stable/velero/templates/crds.yaml (hunk 1 above)
Comment (jmapro) [reply stable/velero/templates/crds.yaml]: Fixded in https://github.com/helm/charts/pull/19042/commits/8d01cb373607144576f32369684e5e5c058622f5

**Review by scottrigby — APPROVED** [review]

**Review by scottrigby — APPROVED** [review]
/lgtm

**Pull request merged**